README.md

```diff
@@ -74,9 +74,10 @@ Setting up DKIM is highly recommended to reduce the chance your emails ending up
 First you need to generate a private and public key for DKIM:
 
 ```bash
-openssl genrsa -out dkim.key 1024
+openssl genrsa -traditional -out dkim.key 1024
```

Comment:
@nguyenkims since dkimpy v1.1.0, [PKCS#8 is supported](https://git.launchpad.net/dkimpy/commit/?id=20d9ca1e1c2ce1fe4bb4b8b52c423de816aaebd7). What about you increase the minimum version instead?

Reply:
Do you know if it's compatible with the old key that people generated? I want to make sure that when people upgrade SL, they don't have any issue.

Reply:
It's a [try/except](https://git.launchpad.net/dkimpy/diff/dkim/crypto.py?id=20d9ca1e1c2ce1fe4bb4b8b52c423de816aaebd7) trying the old key standard first so still compatible :D.